src/main/java/org/ozwillo/dcexporter/model/DcModelMapping.java

```diff
@@ -38,6 +38,10 @@
     private String notes;
     private List<CkanTag> tags;
     private String license;
+    @JsonProperty
+    @NotNull
+    @NotEmpty
+    private String organization;
```

Comment:
- the @JsonProperty annotation doesn't bring anything (there are some on existing fields but it should not)
- adding validation annotations is a good thing, just don't forget the migration part as existing model mappings won't have an organization